### PR TITLE
KAS-3575: Legacy support report generation

### PR DESCRIPTION
**Pull request by sergiofenoll**
https://github.com/kanselarij-vlaanderen/publication-report-service/pull/6

### CODE REVIEW

File: app/components/publications/overview/reports/generate-report-modal.js [hunk 1]
```diff
@@ -198,7 +198,7 @@ export default class GenerateReportModalComponent extends Component {
       'filter[top-concept-schemes][:uri:]':
         CONSTANTS.CONCEPT_SCHEMES.BELEIDSDOMEIN,
       'filter[:has-no:broader]': true, // only top-level government-domains
-      'filter[deprecated]': false,
+      ...((this.publicationYear >= 2022) && { 'filter[deprecated]': false }), // Exclude deprecated government domains when generating modern reports
```
Comment (erikap) [inline app/components/publications/overview/reports/generate-report-modal.js]: Can we define the magic value `2022` in the config file and import it here?

Comment (erikap) [reply app/components/publications/overview/reports/generate-report-modal.js]: What is considered a 'modern' report?  One that is generated after the codelist of beleidsdomein has been finalized?  (trying to find a meaningful name for the variable in the config file)

Comment (sergiofenoll) [reply app/components/publications/overview/reports/generate-report-modal.js]: 2022-03-02 is when OVRB started marking beleidsdomeinen using the current list (which only contains the non-deprecated beleidsdomeinen). Don't know if that makes coming up with a name easier.

Comment (erikap) [reply app/components/publications/overview/reports/generate-report-modal.js]: `PUBLICATIONS_IN_KALEIDOS_START_DATE` ?  :woman_shrugging:

**Pull request merged**